docs/strategies/python/pipenv.md

```diff
@@ -20,10 +20,320 @@ in the top-level `sources`.
+                "installed_version": "0.13.1",
+                "required_version": ">=0.12,<1.0.0a1",
+                "dependencies": [
+                    {
+                        "key": "importlib-metadata",
+                        "package_name": "importlib-metadata",
+                        "installed_version": "4.6.0",
+                        "required_version": ">=0.12",
+                        "dependencies": [
+                            {
+                                "key": "typing-extensions",
+                                "package_name": "typing-extensions",
+                                "installed_version": "3.10.0.0",
+                                "required_version": ">=3.6.4",
+                                "dependencies": []
+                            },
+                            {
+                                "key": "zipp",
+                                "package_name": "zipp",
+                                "installed_version": "3.4.1",
+                                "required_version": ">=0.5",
+                                "dependencies": []
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "key": "py",
+                "package_name": "py",
+                "installed_version": "1.10.0",
+                "required_version": ">=1.8.2",
+                "dependencies": []
+            },
+            {
+                "key": "toml",
+                "package_name": "toml",
+                "installed_version": "0.10.2",
+                "required_version": "Any",
+                "dependencies": []
+            }
+        ]
+    },
+    {
+        "key": "requests",
+        "package_name": "requests",
+        "installed_version": "2.25.1",
+        "required_version": "2.25.1",
+        "dependencies": [
+            {
+                "key": "certifi",
+                "package_name": "certifi",
+                "installed_version": "2021.5.30",
+                "required_version": ">=2017.4.17",
+                "dependencies": []
+            },
+            {
+                "key": "chardet",
+                "package_name": "chardet",
+                "installed_version": "4.0.0",
+                "required_version": ">=3.0.2,<5",
+                "dependencies": []
+            },
+            {
+                "key": "idna",
+                "package_name": "idna",
+                "installed_version": "2.10",
+                "required_version": ">=2.5,<3",
+                "dependencies": []
+            },
+            {
+                "key": "urllib3",
+                "package_name": "urllib3",
+                "installed_version": "1.26.6",
+                "required_version": ">=1.21.1,<1.27",
+                "dependencies": []
+            }
+        ]
+    }
+]
+```
+
+Final graph from our analysis:
+
+![pipenv-graph](pipenv-graph.svg)
```

Comment:
Should we standardize the visual representation of graph in the documentation? I think poetry, mix, and dart has different visual style.